packages/components/src/disabled/README.md

```diff
@@ -49,3 +49,28 @@ function CustomButton() {
 	);
 }
 ```
+
+A component can be conditionally wrapped in a `<Disabled>` with the help of `<Disableable>`.
+
+```jsx
+import { Button, Disableable, TextControl } from '@wordpress/components';
+
+const MyDisabled = withState( {
+	isDisabled: true,
+} )( ( { isDisabled, setState } ) => {
+	const toggleDisabled = () => {
+		setState( ( state ) => ( { isDisabled: ! state.isDisabled } ) );
+	};
+
+	return (
+		<div>
+			<Disableable disabled={ isDisabled }>
+				<TextControl label="Input" onChange={ () => {} } />
+			</Disableable>
+			<Button isPrimary onClick={ toggleDisabled }>
+				Toggle Disabled
+			</Button>
+		</div>
```

Comment:
It might be worth adding this example to the Storybook for Disabled 🙂

Reply:
Good call! I've added an example in https://github.com/WordPress/gutenberg/pull/26730/commits/2daf41f2dd5a4f202a4b60f6f86ddb688838ab30